website/docs/reference/change-requests.mdx

```diff
@@ -10,171 +10,104 @@ import VideoContent from '@site/src/components/VideoContent.jsx';
-
-The skip change requests permission was designed to make it possible to quickly turn something off in the event that a feature release didn't go as expected or was causing issues.
-
-The skip change requests permission does **not** grant any other permissions, so to be allowed to do things as enabling/disabling a flag, the user will still need the explicit permissions to do that too.
-
-In the UI non-admin users with **skip change requests** permission and explicit permission to perform the actual action will be able to make changes directly without change requests.
-
-Admin users will always see the change request UI so that they can test the change request flow. Admin users can however self-approve and self-apply their own changes.
-
-## Change Request for segments
+## Change requests for segments
 
 :::note Availability
 
 **Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `5.4+`
 
 :::
 
-Changes to project [segments](segments) (as opposed to global segments) also go through the change request process. This is to prevent a backdoor in the change request process.
-
-Since projects segments are not environment specific and change requests are always environment specific we allow to attach segment change to any environment with change requests enabled.
-When you make changes though the Change Request UI it will automatically select first environment with change requests enabled, giving priority to [production](environments#environment-types) environments.
-
-Changes to segments can be only circumvented by admin users through the API calls.
-
-## Change Request Preview Playground
-
-To verify that a change request is correct, you can preview the result of change request's application in a change request  [playground](playground). 
-
-![Change request preview](/img/change-request-preview.png)
-
-From the change request overview page, go to the corresponding [playground](playground) and evaluate all your flags in the project and environment that your change request applies too.
-
-![Change request playground evaluation](/img/change-request-playground-evaluation.png)
+Changes to project [segments](/reference#segments), unlike global segments, also go through the change request process.
 
-[Unleash context](playground#the-unleash-context) can be adjusted in the same way as in a regular playground, but the project and environment cannot be changed as they are derived from the change request itself.
-Once the evaluation results confirm the changes in your change request are correct, go back to the change request overview and proceed with the approval or rejection.
+Because project segments are not environment-specific while change requests are, Unleash allows you to attach segment changes to any environment where change requests are enabled. By default, Unleash selects the first available environment with change requests enabled, prioritizing [production](/reference/environments#environment-types) environments.
```

Comment:
"Because project segments are not environment-specific while change requests are," is a bit of a mouthful. Can we simplify it / shorten it / split it into two sentences?

```suggestion
While change requests are environment-specific, project segments are not. For this reason, Unleash allows you to attach segment changes to any environment where change requests are enabled. By default, Unleash selects the first available environment with change requests enabled, prioritizing [production](/reference/environments#environment-types) environments.
```